src/reporter/query_1T1E1A.py

```diff
@@ -57,16 +57,24 @@ def query_1T1E1A(attr_name,   # In Path
         return msg, 500
 
     if entities:
+        if len(entities) > 1:
+            import warnings
+            warnings.warn("Not expecting more than one result for a 1T1E1A.")
+
         if aggr_method:
```

Comment:
may be good to have a warning or error if aggr_period is used but not aggr_method

Reply:
Agree, and we have it :) 

https://github.com/smartsdk/ngsi-timeseries-api/blob/bc8d0bcb97b3e3d2709649cc34d7add9240a1d4c/src/reporter/tests/test_1T1E1A.py#L110